examples/simple-server.rs

```diff
@@ -5,7 +5,7 @@ use simple_server::Server;
 fn main() {
     let server = Server::new(|request, mut response| {
         println!("Request received. {} {}", request.method(), request.uri());
-        response.body("Hello Rust!".as_bytes()).unwrap()
+        Ok(response.body("Hello Rust!".as_bytes())?)
```

Comment:
do we love this syntax?

Reply:
Are you referring to `Ok`, the `?`, or `as_bytes`

Reply:
the `Ok`, `?`